user-guide/toc.yml

```diff
@@ -2134,6 +2134,64 @@
           topicUid: Configuring_custom_apps
         - name: Deleting a custom application
           topicUid: Deleting_custom_apps
+    - name: DataMiner Object Models (DOM)
+      topicUID: DOM
+      items:
+        - name: DOM objects
+          topicUid: DOM_objects
+          items:
+            - name: SectionDefinition object
+              topicUID: DOM_SectionDefinition
+            - name: DomDefinition object
+              topicUID: DomDefinition
```

Comment:
If UIDs do not contain the term "Dom" anywhere, I've added that prefix just to be safe, so that the same UID does not get used somewhere else as well. For "DomDefinition", for example, that risk doesn't exist, while "SectionDefinition" could also get used in the context of the Jobs app or even elsewhere.